translations/monero-core_pt-pt.ts

```diff
@@ -1221,155 +1221,155 @@ O ficheiro cache antigo da carteira será renomeado e poderá ser reposto depois
-        <translation type="unfinished"></translation>
+        <translation>Este conjunto de transacções é conhecido como estando gasto, e não devem ser usados em futuras ring signatures.</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="135"/>
         <source>You should only have to load a file when you want to refresh the list. Manual adding/removing is possible if needed.</source>
-        <translation type="unfinished"></translation>
+        <translation>Apenas deve carregar um ficheiro quando pretende refazer a lista. Aidiconar/remover manualmente é possivel se necessário.</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="147"/>
         <source>Please choose a file to load blackballed outputs from</source>
-        <translation type="unfinished"></translation>
+        <translation>Por favor escolha um ficheiro para carregar uma lista negra de endereços</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="163"/>
         <source>Path to file</source>
-        <translation type="unfinished"></translation>
+        <translation>Directorio para o ficheiro</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="165"/>
         <source>Filename with outputs to blackball</source>
-        <translation type="unfinished"></translation>
+        <translation>Nome do ficheiro com lista negra de endereços</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="178"/>
         <source>Browse</source>
-        <translation type="unfinished"></translation>
+        <translation>Navegar</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="189"/>
         <source>Load</source>
-        <translation type="unfinished"></translation>
+        <translation>Carregar</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="205"/>
         <source>Or manually blackball/unblackball a single output:</source>
-        <translation type="unfinished"></translation>
+        <translation>Ou manualmente adicionar/remover da lista negra um único endereço:</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="206"/>
         <source>Paste output public key</source>
-        <translation type="unfinished"></translation>
+        <translation>Cole o endereço publico</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="220"/>
         <source>Blackball</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Adicionar</translation>
```

Comment:
is this left 'unfinished' on purpose?

Reply:
Yes, I am not sure if this translation is the correct aproach, I need to see where this stay on the GUI to be sure

Reply:
Build is broken right now, you should cherrypick monero-project/monero#3745 to be able to compile. Btw that "blackball" is basically "blacklist". Check [this section of the guide](https://github.com/monero-ecosystem/monero-GUI-guide/blob/master/monero-GUI-guide.md#53-shared-ringdb) to see where those buttons are